src/styles/_catalog.scss

```diff
@@ -1,22 +1,21 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
-
-  max-width: 944px;
-  margin: 0 auto;
+  display: grid;
+  grid-template-columns: repeat(auto-fill, $card-width);
+  gap: 48px;
+  justify-content: center;
+  justify-items: center;
 
   &__card {
     display: flex;
     justify-content: center;
     align-items: center;
-    width: 200px;
+    width: $card-width;
     height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
   }
 
-  &__card:not(:nth-child(4n)) {
-    margin-right: 48px;
+  @media (min-width: 1024px) {
+    grid-template-columns: repeat(4, $card-width);
   }
```

Comment:
There should be three columns **starting** at 768px
There should be two columns **starting** at 488px
![image](https://user-images.githubusercontent.com/37161315/195094704-c9d319b0-df11-492c-b7f7-64ee1aa69a79.png)
![image](https://user-images.githubusercontent.com/37161315/195094829-4f83172a-78ca-4806-83d7-6dff9ce9f6e3.png)
Create corresponding media rules